tests/Testcontainers.Tests/Unit/Containers/Unix/TestcontainersContainerTest.cs

```diff
@@ -3,6 +3,7 @@ namespace DotNet.Testcontainers.Tests.Unit.Containers.Unix
   using System;
   using System.Globalization;
   using System.IO;
+  using System.Linq;
```

Comment:
Remove unnecessary using.

Reply:
Fixed in [0a4072c](https://github.com/testcontainers/testcontainers-dotnet/pull/480/commits/0a4072cfc06a960bfc16130676a0a01e439e85cd)